Policies/Kubernetes/allowed-external-ips/rego/src.rego

```diff
@@ -0,0 +1,11 @@
+package k8sazureexternalips
+
+violation[{"msg": msg}] {
+  input.review.kind.kind == "Service"
+  input.review.kind.group == ""
+  allowedIPs := {ip | ip := input.parameters.allowedIPs[_]}
+  externalIPs := {ip | ip := input.review.object.spec.externalIPs[_]}
+  forbiddenIPs := externalIPs - allowedIPs
+  count(forbiddenIPs) > 0
+  msg := sprintf("service has forbidden external IPs: %v", [forbiddenIPs])
```

Comment:
Can you change the message to include the list of allowed IPs? `"Service has forbidden external IPs: %v, Allowed IPs: %v"`

Reply:
I am not sure it is by design that the allowed IPs are not shown. Let's check it with @ritazh.

Reply:
I think for now we should follow what upstream https://github.com/open-policy-agent/gatekeeper-library/blob/master/library/general/externalip/template.yaml#L32 is doing for this policy. I can see cases where displaying the allowed ip might not be a good idea. Let's revisit this if a user asks for the allowed IP to be returned.